packages/drivers/odsp-driver/src/contracts.ts

```diff
@@ -7,6 +7,20 @@ import * as api from "@fluidframework/protocol-definitions";
 import { HostStoragePolicy } from "@fluidframework/odsp-driver-definitions";
 import { ISnapshotContents } from "./odspUtils";
 
+/** https://portal.microsofticm.com/imp/v3/incidents/details/308931013/home
+ * The commits property was removed from protocol-definitions but in order to support back compat, we will
+ * temporarily add back in this local structure in order to upload the snapshot to support rolling back to 0.58.
+ * Notice this entire interface will be removed once the backward compatibility is not required anymore.
+*/
+export interface ISnapshotTreeEx {
+    id?: string;
+    blobs: { [path: string]: string; };
+    commits: { [path: string]: string; };
```

Comment:
Is this the same code which you merged in release branch? There we extended the ISnapshotTree interface to add commits. Are you cherry picking wrong commit?